src/main/utils.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns the major version of electron
+ */
+export function getElectronVersion(): { major: number; minor: number; patch: number } {
+  const versions = process.versions.electron.split('.').map(val => parseInt(val, 10));
```

Comment:
This will not work for all currently used semver strings (ie. `v10.0.0-beta.11`)

This should be more reliable:
```typescript
const { groups } = /(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)/.exec(process.versions.electron);
return groups;
```